app/views/ops/_all_tabs.html.haml

```diff
@@ -153,7 +147,7 @@
           = miq_tab_content("diagnostics_workers", @sb[:active_tab]) do
             = render :partial => "diagnostics_workers_tab"
         = miq_tab_content("diagnostics_collect_logs", @sb[:active_tab]) do
-          = render :partial => "diagnostics_collect_logs_tab"
+          = render :partial => "diagnostics_collect_logs_tab" ## delete this and the solo page becomes empty
```

Comment:
Can you explain this more? I expected the entire tab content and the partial itself to be deleted.

Reply:
Yeah! So i asked around about wether or not this page was still needed to be kept since it wasn't explicitly a tab but still showed content that belonged in the tab

![image](https://user-images.githubusercontent.com/29209973/164741673-10ad278e-4fbc-490f-a1b4-76e7361c184c.png)

since i wasn't sure if it also was to be deleted so I've been waiting for a response (which you have basically just given me so thank you and ill be deleting this very very soon)

Reply:
Since we're 🗑️  file depot, which is a log depot, and collect logs uses log depot, I think this can also be 🗑️